baselines/diabetic_retinopathy_detection/jax_finetune_deterministic.py

```diff
@@ -14,147 +14,89 @@
 # limitations under the License.
 
 """Finetuning."""
+import tensorflow as tf
+
+tf.config.experimental.set_visible_devices([], 'GPU')
+tf.config.experimental.set_visible_devices([], 'TPU_SYSTEM')
+tf.config.experimental.set_visible_devices([], 'TPU')
+
 import functools
 import itertools
 import multiprocessing
 import numbers
 import os
 import time
 
+import flax
```

Comment:
Can we revert the import reordering? When merging #735, the import order change failed the linter.

https://github.com/google/uncertainty-baselines/commit/62843fb31276d8020adda6d295edeab71ffa0b4b